alertmanager/values.yaml

```diff
@@ -0,0 +1,31 @@
+## NOTE: This is still a work in progress. Contributions welcome!
+##
+## For Thanos environments, each of the below expressions are run against every
```

Comment:
This is automatic or requires some additional config for thanos?

Reply:
Automatic for Thanos! FedETL environments would need to configure these Alerts on each individual Prometheus deployment.